kani-driver/src/args/mod.rs

```diff
@@ -281,6 +281,10 @@ pub struct VerificationArgs {
     #[arg(long, hide = true)]
     pub print_llbc: bool,
 
+    /// Timeout for CBMC command in seconds
```

Comment:
Should we make this unstable for now? Require `-Z unstable-options`

Reply:
Also, I was wondering if we could accept an optional suffix like the `timeout` command.

Reply:
> Should we make this unstable for now? Require `-Z unstable-options`

Sure.

> Also, I was wondering if we could accept an optional suffix like the `timeout` command.

Yes, that would be convenient.

Reply:
> > Also, I was wondering if we could accept an optional suffix like the `timeout` command.
> 
> Yes, that would be convenient.

Actually, can I do this in a follow-up PR? It's not difficult to implement, but will require a number of tests to cover the different scenarios (including negative tests).

Reply:
Never mind; added support for a unit suffix in https://github.com/model-checking/kani/pull/3649/commits/c96ea92a87dec18a485be6ba36dfd6b3bf6ca804